### ISSUE TITLE
Train's report prints infinite training loss

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#238]
When disply step was set to 10 the WER report began as follows:
```
2016-11-25 17:41:55 STARTING Epoch 0009
2016-11-25 17:41:55 Training model...
2016-11-25 18:29:38 Training loss=inf avg_cer=0.377731130 WER: 0.828431
```

**Comment by kdavis-mozilla** [comment]
Looks like this was solved by setting `learning_rate=0.0001` the old `learning_rate` was too large.
